src/app/tests/suites/certification/Test_TC_SMCO_2_4.yaml

```diff
@@ -0,0 +1,601 @@
+          PIXIT.SMCO.TEST_EVENT_TRIGGER for Hardware Fault Alert Test Event"
+      PICS: DGGEN.S.C00.Rsp
+      cluster: "General Diagnostics"
+      endpoint: 0
+      command: "TestEventTrigger"
+      arguments:
+          values:
+              - name: "EnableKey"
+                value: TEST_EVENT_TRIGGER_KEY
+              - name: "EventTrigger"
+                value: TEST_EVENT_TRIGGER_HARDWARE_FAULT_ALERT
+
+    - label:
+          "TH waits for a report of HardwareFaultAlert attribute from DUT with a
+          timeout of 300 seconds"
+      PICS: SMOKECO.S.A0006
+      command: "waitForReport"
+      attribute: "HardwareFaultAlert"
+      timeout: 300
+      response:
+          value: 1
+          constraints:
+              type: boolean
+
+    - label: "TH reads ExpressedState attribute from DUT"
+      PICS: SMOKECO.S.A0000
+      command: "readAttribute"
+      attribute: "ExpressedState"
+      response:
+          value: 5
+          constraints:
+              type: enum8
+
+    - label: "TH reads HardwareFault event from DUT"
+      PICS: SMOKECO.S.E03
+      command: "readEvent"
+      event: "HardwareFault"
+      eventNumber: eventNumber + 4
+      response:
+          value: {}
+
+    - label:
+          "TH sends TestEventTrigger command to General Diagnostics Cluster on
+          Endpoint 0 with EnableKey field set to
+          PIXIT.SMCO.TEST_EVENT_TRIGGER_KEY and EventTrigger field set to
+          PIXIT.SMCO.TEST_EVENT_TRIGGER for Hardware Fault Alert Test Event
+          Clear"
+      PICS: DGGEN.S.C00.Rsp
+      cluster: "General Diagnostics"
+      endpoint: 0
+      command: "TestEventTrigger"
+      arguments:
+          values:
+              - name: "EnableKey"
+                value: TEST_EVENT_TRIGGER_KEY
+              - name: "EventTrigger"
+                value: TEST_EVENT_TRIGGER_HARDWARE_FAULT_ALERT_CLEAR
+
+    - label:
+          "TH waits for a report of HardwareFaultAlert attribute from DUT with a
+          timeout of 300 seconds"
+      PICS: SMOKECO.S.A0006
+      command: "waitForReport"
+      attribute: "HardwareFaultAlert"
+      timeout: 300
+      response:
+          value: 0
+          constraints:
+              type: boolean
+
+    - label: "TH reads ExpressedState attribute from DUT"
+      PICS: SMOKECO.S.A0000
+      command: "readAttribute"
+      attribute: "ExpressedState"
+      response:
+          value: 0
+          constraints:
+              type: enum8
+
+    - label: "TH reads AllClear event from DUT"
+      PICS: SMOKECO.S.E0a
+      command: "readEvent"
+      event: "AllClear"
+      eventNumber: eventNumber + 5
+      response:
+          value: {}
+
+    - label: "TH subscribes to EndOfServiceAlert attribute from DUT"
+      PICS: SMOKECO.S.A0007
+      command: "subscribeAttribute"
+      attribute: "EndOfServiceAlert"
+      minInterval: 3
+      maxInterval: 30
+      response:
+          value: 0
```

Comment:
Test plan in https://github.com/CHIP-Specifications/chip-test-plans/pull/2596/files does not seem to specify a value here, but probably should....  Test plan issue, please.

Reply:
https://github.com/CHIP-Specifications/chip-test-plans/issues/3110